src/idvc/__init__.py

```diff
@@ -12,3 +12,7 @@
 
 #   Author: Laura Murgatroyd (UKRI-STFC)
 #   Author: Edoardo Pasca (UKRI-STFC)
+try:
+    import ccpi.viewer
```

Comment:
The viewer has been moved to https://github.com/TomographicImaging/CILViewer. Can we update this?

Reply:
Apparently the conda is still called ccpi so my comment is not relevant.